README.md

```diff
@@ -2,14 +2,14 @@
 
 [![Build Status](https://github.com/SeismoJulia/SeisBase.jl/actions/workflows/ci.yml/badge.svg)](https://github.com/SeismoJulia/SeisBase.jl/actions/workflows/ci.yml)  
 
-<!-- [![codecov](https://codecov.io/gh/jpjones76/SeisIO.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/jpjones76/SeisIO.jl)[![Coverage Status](https://coveralls.io/repos/github/jpjones76/SeisIO.jl/badge.svg?branch=master)](https://coveralls.io/github/jpjones76/SeisIO.jl?branch=master) [![Documentation Status](https://readthedocs.org/projects/seisio/badge/?version=latest)](https://seisio.readthedocs.io/en/latest/?badge=latest) -->
+<!-- [![codecov](https://codecov.io/gh/jpjones76/SeisBase.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/jpjones76/SeisBase.jl)[![Coverage Status](https://coveralls.io/repos/github/jpjones76/SeisBase.jl/badge.svg?branch=master)](https://coveralls.io/github/jpjones76/SeisBase.jl?branch=master) [![Documentation Status](https://readthedocs.org/projects/SeisBase/badge/?version=latest)](https://SeisBase.readthedocs.io/en/latest/?badge=latest) -->
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 
 A minimalist, platform-agnostic package for univariate geophysical data.
-This version is a follow up from SeisIO.jl (https://github.com/jpjones76/SeisIO.jl). It is intended to be an stable community maintained package.
+This version is a follow up from SeisBase.jl (https://github.com/jpjones76/SeisBase.jl). It is intended to be an stable community maintained package.
 
-## Installation | [Documentation](http://seisio.readthedocs.org)
-From the Julia prompt, type: `] add SeisIO`; (Backspace); `using SeisIO`
+## Installation | [Documentation](http://SeisBase.readthedocs.org)
+From the Julia prompt, type: `] add SeisBase`; (Backspace); `using SeisBase`
```

Comment:
```suggestion
From the Julia prompt, type: `] add https://github.com/SeismoJulia/SeisBase.jl`; (Backspace); `using SeisBase`
```
this can be changed once the package is published.